### PR TITLE
Update to use php7.3

### PR DESCRIPTION
**Pull request by coffeeneer**
I am using this package in Laravel. When updating to the newest version I ran into issue with the support for Guzzle 7.
This update includes the update to Guzzle 7 and the newest version of PHPUnit which will be supported until February 2022.

If I have some spare time I might also update the client itself to use the php 7 supported typing (return types and argument types).

### CODE REVIEW

**Comment by coffeeneer** [comment]
The current support for PHPUnit:
Major Version | PHP Compatibility | Initial Release | Support
-- | -- | -- | --
PHPUnit 9 | PHP 7.3, PHP 7.4 | February 7, 2020 | Support ends on February 4, 2022
PHPUnit8 | PHP 7.2, PHP 7.3, PHP 7.4 | February 1, 2019 | Support ends on February 5, 2021

The minimum PHP version requirement for Guzzle & is 7.2, so support for 7.2 could be an option as well when using PHPUnit8. Taking into account that the PHPUnit 8's support is ending soon-ish.

File: composer.lock [hunk 1]
```diff
@@ -0,0 +1,2337 @@
+{
```
Comment (m1x0n) [inline composer.lock]: Since it's a package we don't need `composer.lock` file here. Could you please remove it?

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 .idea/
 vendor/
-composer.lock
```
Comment (m1x0n) [inline .gitignore:3]: Could you please move composer.lock back to ignore?

**Review by m1x0n — CHANGES_REQUESTED** [review]
@dodedodo33 I left some comments regarding composer.lock file. Besides good job man!

File: composer.lock (hunk 1 above)
Comment (coffeeneer) [reply composer.lock]: Ah yes, I'm used to commiting that for projects.

**Review by m1x0n — APPROVED** [review]

**Comment by coffeeneer** [comment]
@m1x0n Thanks for your quick action! And thanks for developing and publishing this package in the first place :)

**Pull request merged**